### DISCUSSION TITLE
Fachada e agregador de dados como Dados Abertos e Portal Transparência

### DISCUSSION THREAD

**Discussion by tupizz** [BrasilAPI/BrasilAPI#249] [Ideas]
Seria interessante, afim de trazer à sociedade uma visão mais transparente, e livre de conceitos técnicos do mundo político, o panorama geral de cada um dos deputados eleitos. Informações como:

1. Gastos parlamentares de uma forma bem simples de entender;
2. Últimos projetos em que o deputado **se mostrou a favor ou contra**;
3. Ter um feed de notícias a respeito de cada um desses deputados;
4. Alimentar um repositório (instagram ou facebook) com as informações mais quentes desse mundo político.

**Links:**

[Portal Transparência - Despesas](http://www.portaltransparencia.gov.br/download-de-dados/despesas-execucao)
[Dados Abertos - Swagger
](https://dadosabertos.camara.leg.br/swagger/api.html)

**Comment by valdirmendesdev** [comment]
@tupizz Tadeu, o que você pensou sobre isso? Qual seria sua ideia? Eu já tive algumas ideias sobre apresentar essas informações um pouco mais "mastigadas", mas não consegui chegar num modelo que eu entendia como viável, sabe?

**Comment by tupizz** [comment]
@valdirmendesdev Hmm.. A priori pensei em algo mais simples como acompanhar quais projetos cada deputado votou, por exemplo para ver se ele se manteve constante com as suas promessas de campanha, ou se "se vendeu" com o passar dos anos. 
Os gastos parlamentares poderia ser algo amais para ver se ele está utilizando o dinheiro público para gastos pífios. E com isso ser um mecanismo de cobrança da população com esse deputado.
E por fim, o feed de notícias pensei em coisas bem legais, mas para simplificar a história, num primeiro momento poderíamos pegar:
- O feed do twitter de cada deputado
- O que as pessoas estão falando dele

Acredito que de início seria isso. Lembrando que é uma sugestão, se você achar interessante e topar podemos pensar um projeto a parte nosso com contribuições mútuas. O que acha?

Valeu!

**Comment by valdirmendesdev** [comment]
Sensacional mano, achei a ideia fantástica! O @filipedeschamps teló vai ficar triste conosco, mas acho que seria interessante! Há pouco tempo estava lendo sobre avaliação emocional dos tweets e acho que tem muita coisa legal que dá para fazer! Animo demais!

**Comment by willian2s** [comment]
Achei sensacional a ideia, ainda tô no início da minha jornada, mas acredito que seria uma boa base de aprendizado e que poderia ajudar se quiserem
